src/components/Hero/_Hero.scss

```diff
@@ -1,13 +1,17 @@
 .hero {
-  // NOTE: image path is relative to where the _Hero.scss partial is imported in index.scss
-  background: url(./images/heading-banner-bg.png);
```

Comment:
This import wasn't working, and since we discussed removing this in the last UX meeting, I just went for it. Will discuss / confirm with Jenn before merge.

Reply:
I have confirmed that we *can* add images in css, but with paths relative to the file in question, not relative to `index.scss`. Removing this image anyway because we wanted to do that eventually anyway.